asciidoc-confluence-publisher-client/src/test/java/org/sahli/asciidoc/confluence/publisher/client/ConfluencePublisherTest.java

```diff
@@ -345,14 +482,11 @@ public void publish_metadataWithExistingPageAndAttachmentWithSameAttachmentConte
         ConfluenceRestClient confluenceRestClientMock = mock(ConfluenceRestClient.class);
```

Comment:
Similar: is this scenario not already tested by `publish_whenAttachmentsHaveSameContentHash_doesNotUpdateAttachments` ?